specification/compute/resource-manager/Microsoft.Compute/stable/2021-03-01/cloudService.json

```diff
@@ -0,0 +1,2684 @@
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CloudServiceRoleProfileProperties"
+          }
+        }
+      }
+    },
+    "CloudServiceVaultCertificate": {
+      "description": "Describes a single certificate reference in a Key Vault, and where the certificate should reside on the role instance.",
+      "type": "object",
+      "properties": {
+        "certificateUrl": {
+          "description": "This is the URL of a certificate that has been uploaded to Key Vault as a secret.",
+          "type": "string"
+        }
+      }
+    },
+    "CloudServiceVaultSecretGroup": {
+      "description": "Describes a set of certificates which are all in the same Key Vault.",
+      "type": "object",
+      "properties": {
+        "sourceVault": {
+          "$ref": "#/definitions/SubResource",
+          "description": "The relative URL of the Key Vault containing all of the certificates in VaultCertificates."
+        },
+        "vaultCertificates": {
+          "description": "The list of key vault references in SourceVault which contain certificates.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CloudServiceVaultCertificate"
+          }
+        }
+      }
+    },
+    "CloudServiceOsProfile": {
+      "description": "Describes the OS profile for the cloud service.",
+      "type": "object",
+      "properties": {
+        "secrets": {
+          "description": "Specifies set of certificates that should be installed onto the role instances.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CloudServiceVaultSecretGroup"
+          }
+        }
+      }
+    },
+    "LoadBalancerFrontendIPConfigurationProperties": {
+      "description": "Describes a cloud service IP Configuration",
+      "type": "object",
+      "properties": {
+        "publicIPAddress": {
+          "$ref": "#/definitions/SubResource",
+          "description": "The reference to the public ip address resource."
+        },
+        "subnet": {
+          "$ref": "#/definitions/SubResource",
+          "description": "The reference to the virtual network subnet resource."
+        },
+        "privateIPAddress": {
+          "description": "The virtual network private IP address of the IP configuration.",
+          "type": "string"
+        }
+      }
+    },
+    "LoadBalancerFrontendIPConfiguration": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "The name of the resource that is unique within the set of frontend IP configurations used by the load balancer. This name can be used to access the resource.",
+          "type": "string"
+        },
+        "properties": {
+          "$ref": "#/definitions/LoadBalancerFrontendIPConfigurationProperties",
+          "description": "Properties of load balancer frontend ip configuration."
+        }
+      },
+      "required": [
```

Comment:
Adding a required property to an existing object is a breaking change. We do not want to have any breaking changes comparing the latest Preview release with the first Stable release. 
I think you're also trying to edit this in the preview version. That Preview needs to release and release the SDK before this change can be merged.

Reply:
resolved as per email discussion